treeherder/webapp/api/bugzilla.py

```diff
@@ -15,7 +15,7 @@ def create_bug(self, request):
         """
         Create a bugzilla bug with passed params
         """
-        if settings.BZ_API_KEY is None:
+        if settings.BUGFILER_API_KEY is None:
             return Response({"failure": "Bugzilla API key not defined. This shouldn't happen."},
```

Comment:
Whilst we're here, let's change `"Bugzilla API key not defined. This shouldn't happen."` to `"Bugzilla API key not set!"`.

(It can happen when developing locally)